demo/paging/paging-virtual.component.ts

```diff
@@ -27,7 +27,7 @@ import {Page} from "./model/page";
         [headerHeight]="50"
         [scrollbarV]="true"
         [footerHeight]="50"
-        [rowHeight]="50"
+        [rowHeight]="100"
```

Comment:
Every change on demo/paging/paging-virtual.component.ts should have not been pushed through. I started using this demo to try out row grouping functionality before I actually created the exclusive demo.